### PR TITLE
Use server date year as copyright end year

### PR DESCRIPTION
**Pull request by stefan-muc**
To not have to update it every year or getting it wrong as in 1.6.8 simply use the year of the server date as copyright end year

### CODE REVIEW

**Comment by alecpl** [comment]
I think we should just remove the year(s), as we did in source file headers some time ago. Copyright years do not provide much value. And the info can be derived from git history.

**Comment by pabzm** [comment]
@stefan-muc Thank you for your contribution! As you see it invoked a thought to solve the topic differently. Would you be willing to remove the years instead of changing them? Then we could merge this!

**Comment by stefan-muc** [comment]
Sure I'm willing! I just thought that simply removing it can be done by the team itself - but here you are with a changed pull request.

**Comment by pabzm** [comment]
Of course we could have done it ourselves, but I didn't want to steal the opportunity of a contribution from you. :)

Thank you!

File: installer/index.php [hunk 1]
```diff
@@ -164,7 +164,7 @@
 </div>
 
 <div id="footer">
-  Installer by the Roundcube Dev Team. Copyright &copy; 2008-2021 – Published under the GNU Public License;&nbsp;
+  Installer &copy; by The Roundcube Dev Team – Published under the GNU Public License;&nbsp;
```
Comment (pabzm) [inline installer/index.php:167]: Could you please re-add the word "Copyright" (since is used in the other places as well in combination with the symbol).

**Review by pabzm — CHANGES_REQUESTED** [review]

File: installer/index.php (hunk 1 above)
Comment (stefan-muc) [reply installer/index.php]: Sorry, somehow I missed this. Re-added now!

**Review by pabzm — APPROVED** [review]
Thank you! 👍

**Comment by pabzm** [comment]
@alecpl I'll merge this on Monday unless you object. (The CI errors are unrelated, don't happen locally neither.)

**Pull request merged**